boards/others/neorv32/neorv32_neorv32_minimalboot.dts

```diff
@@ -21,6 +22,10 @@
 		pwm-led0 = &pwm_led0;
 		pwm-led1 = &pwm_led1;
 		pwm-led2 = &pwm_led2;
+		sw0 = &btn0;
+		sw1 = &btn1;
+		sw3 = &btn2;
+		sw4 = &btn3;
```

Comment:
4 GPIO resources as in the VHDL file :white_check_mark:
https://github.com/stnolting/neorv32-setups/blob/main/osflow/board_tops/neorv32_UPDuino-v3.0_BoardTop_UP5KDemo.vhd#L60-L62

Reply:
Thank you for reviewing.

The number of GPIOs is correct, except you are referencing the wrong HDL file. This is the correct reference: https://github.com/stnolting/neorv32/blob/99a32e5be9d425eef8e0504b37be332b94647893/rtl/processor_templates/neorv32_ProcessorTop_MinimalBoot.vhd#L27

Reply:
I stand corrected!